api/intake/intakeModel.js

```diff
@@ -0,0 +1,56 @@
+const db = require('../../data/db-config');
+
+const update = (id, changes) => {
+  return db('clients').where({ id }).update(changes, '*');
+};
+
+const updateEducationHistoryByClientId = (id, changes) => {
+  return db('education_histories')
+    .where({ client_id: id })
+    .update(changes, '*');
```

Comment:
Let's do one thing here...
Instead of having an intake model: 
1. Put the model functions for client into the client model.
2. Put the model functions for household into the household model.